swagger.yaml

```
@@ -44,21 +422,255 @@ paths:
+            readOnly: true
+            items:
+              $ref: "#/definitions/ContentItem"
+          creator:
+            description: "The user that created the bundle, taken from the auth token supplied.  This should only be set on initial creation of the bundle."
+            type: string
+            example: "publisher@ons.gov.uk"
+          created_date:
+            description: "The time and date the bundle was created on."
+            type: string
+            format: date-time
+            readOnly: true
+            example: "2025-04-04T07:00:00.000Z"
+          id:
+            type: string
+            description: "The bundle id"
+            readOnly: true
+            example: "9e4e3628-fc85-48cd-80ad-e005d9d283ff"
+          last_updated_by:
+            description: "The user that last updated the bundle, taken from the auth token supplied."
+            type: string
+            example: "publisher@ons.gov.uk"
+          preview_teams:
+            description: "A list of teams who have permissions to view the dataset series in the bundle.  Supplied by AWS cognito."
+            type: array
+            items:
+              type: string
+          publish_date_time:
+            description: "The time and date of publication."
+            type: string
+            format: date-time
+            example: "2025-04-04T07:00:00.000Z"
+          state:
+            description: "The approval status of the overall bundle (TBC - values provided are examples only)."
+            type: string
+            enum:
+              - not_started
+              - in_progress
+              - approved
+            example: "approved"
+          title:
+            type: string
+            description: "The title of the bundle"
+            example: "CPI March 2025"
+          updated_date:
+            description: "The time and date the bundle was last updated."
+            type: string
+            readOnly: true
+            format: date-time
+            example: "2025-04-04T07:00:00.000Z"
+          wagtail_managed:
+            description: "If the bundle creation was initiated by wagtail"
+            type: boolean
+  Contents:
+    description: "A list of contents related to a bundle"
+    type: object
+    allOf:
+      - $ref: "#/definitions/PaginationFields"
+      - type: object
+        properties:
+          items:
+            type: array
+            items:
+              $ref: "#/definitions/ContentItem"
+  ContentItem:
+    description: "A model which holds information about the datasets to be published as part of the bundle"
+    allOf:
+      - type: object
+        properties:
+          dataset_id:
+            type: string
+            description: "The dataset id of the item in the bundle"
+            example: "cpih"
+          edition_id:
+            type: string
+            description: "The edition id of the dataset item in the bundle"
+            example: "march"
+          item_id:
```

As this is the primary ID of this (sub-)resource it should be `id`:

```suggestion
          id:
```

---

This has been changed.